website/src/css/custom.css

```diff
@@ -488,4 +488,9 @@ body {
   border-color: #f86a2b;
   font-size: 0.8em;
   font-weight: 500;
+}
+
+.fixedToc {
+  position: fixed;
+  width: 17em;
```

Comment:
Fixed width value leads to it being cut off. Think it needs a `max-width: 15%` or something also
<img width="1362" alt="image" src="https://github.com/appsmithorg/appsmith-docs/assets/39881648/8758e58d-6ccc-4ed8-b525-dce58a860fbe">